ocfweb/docs/docs/staff/tips/emailtemplates.md

```diff
@@ -0,0 +1,7 @@
+[[!meta title="Email Templates"]]
+
+General email response templates can be found at `templates.ocf.berkeley.edu`.
+
+Some templates have variables that should be filled out before copying the template.
```

Comment:
The template variables will be automatically filled out by entering text into the input boxes above the main text box.